test/test_torch.py

```diff
@@ -7734,6 +7734,20 @@ def test_helper(dim1, dim2, memory_format):
         test_helper((3, 3), (3, 3, 3, 3), torch.channels_last)
         test_helper((3, 3, 3), (3, 3, 3, 3, 3), torch.channels_last_3d)
 
+    def test_dim_order(self):
+        shape = (2, 3, 5, 7)
+
+        self.assertSequenceEqual(torch.empty(shape).dim_order(), (0, 1, 2, 3), seq_type=tuple)
```

Comment:
can you add some test cases where we create slice of tensor across multiple dims
e.g. t = t[:, 2:5, 7:9]...

Reply:
Ah taking a slice doesn't change the strides. So Ok to not have this in the tests